package.json

```diff
@@ -105,12 +105,12 @@
     "string-replace-loader": "^2.2.0",
     "style-loader": "^0.23.1",
     "styled-components": "4.4.1",
-    "terser-webpack-plugin": "^1.1.0",
+    "terser-webpack-plugin": "3.0.1",
     "text-encoding": "^0.7.0",
     "unist-util-visit": "^1.4.0",
     "url-loader": "^1.1.2",
     "url-search-params": "^1.1.0",
-    "webpack-cli": "^3.3.3",
+    "webpack-cli": "3.3.11",
```

Comment:
Is it intentional to lock the webpack-cli versions? By not locking it we probably can catch more issues related to dependency versions.

Reply:
In our internal version we lock all versions so I figured I'd do it here too. Shouldn't matter too much though because we also have package-lock.json? How would we catch more issues related to dependency versions?